core/src/main/java/greencity/webcontroller/ManagementAchievementController.java

```diff
@@ -44,12 +49,19 @@ public class ManagementAchievementController {
     public String getAllAchievement(@RequestParam(required = false, name = "query") String query, Pageable pageable,
         Model model) {
         Pageable paging = PageRequest.of(pageable.getPageNumber(), pageable.getPageSize(), Sort.by("id").descending());
-        PageableAdvancedDto<AchievementVO> allAchievements = query == null || query.isEmpty()
+        PageableAdvancedDto<AchievementDto> allAchievements = query == null || query.isEmpty()
```

Comment:
Instead of use query == null || query.isEmpty(); Use safe from null method from String Utils  .isNotEmpty()